app-emacs/ht/ht-2.3.ebuild

```
@@ -0,0 +1,22 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit elisp
+
+DESCRIPTION="The missing hash table library for Emacs."
+HOMEPAGE="https://github.com/Wilfred/ht.el"
+SRC_URI="https://github.com/Wilfred/ht.el/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}.el-${PV}"
```

Nit, see other quoting remark:
```suggestion
S="${WORKDIR}"/${PN}.el-${PV}
```